Topics/Software_Engineering/Sprint Planning Meeting.md

```diff
@@ -0,0 +1,40 @@
+
+
+## What does a Sprint Planning Meeting Encompass? 
+
+| Scrum Artifacts | Definition |
+| -------- | ------- |
+| Product Backlog  | an ordered list of tasks essential for improving the product and satisfying the stakeholders  |
+| Sprint Backlog | a list of tasks selected from the Product Backlog to be completed during a specific sprint     |
+
+* The meeting is held with the entire Agile team (including the ScrumMaster and Product Owner) and aims to conclude with a set of agreed upon **Product Backlog** items to be completed for the current sprint commitment. 
+* The ScrumMaster is responsible for leading the Planning Meeting and the Product Owner is responsible for detailing the Product Backlog items and ensuring alignment with the product’s goals. 
+* The planning addresses the value the sprint can bring to the stakeholders and the team collaborates to define a **Sprint Goal** before the end of the Sprint Planning meeting. 
+* Upon discussion with the Product Owner, the developers select items from the Product Backlog to include in the **Sprint Backlog** and determine this based on their prior and future Sprint performance. 
+* For each Sprint Backlog item that is selected, the developers on the team decide how to break down and complete tasks for the sprint based on the team’s definition of “Done”.
+
+
+## Integrating Sprint Planning Meetings in CSC301 
+*Consider an example of how a Scrum Planning Meeting can be integrated into our course’s workflow.*
+
+### Before the Planning Meeting:
+After a Product Manager and Scrum Master are chosen in Deliverable 1, consider each deliverable (deliverable 2 to 5) equivalent to a sprint, with each sprint having one dedicated Sprint Planning Meeting. As a ScrumMaster and Product Manager, organize an hour long meeting within the next week after the Deliverable handout is released. Prior to the Scrum Planning Meeting, you both should also read through the handout and identify features/Product Backlog items for the upcoming sprint and add them to the Product Backlog list on Jira (or the team’s chosen product management tool).
+
+### During the Planning Meeting: 
+Start by reminding the team of the upcoming Deliverable and review the definition of “Done”, which in this case would be meeting both the rubric and your partner's requirements. As a team, take this opportunity to brainstorm more tasks that need to be completed based on what your partner has asked and add them into the Product Backlog list.
+
+Discuss how much time would be needed for each Product Backlog item listed on Jira and based on this estimate, determine items for the Sprint Backlog. Following, select items from the Product Backlog and add them into the Sprint Backlog, allow team members to assign themselves to sprint tasks (including yourself) and discuss any further action items as a team. 
+
+Once the Sprint Backlog items are finalized, as the ScrumMaster, call for a group consensus on the plan and conclude the meeting.
```

Comment:
Great job explaining how we can utilize sprints in CSC301 ! 
Perhaps, you can  include a section on what you can do if there is disagreements and how they nay affect the sprint.

Reply:
Great suggestion! Will include this both in the  _What does a Sprint Planning Meeting Encompass?_ and _Integrating Sprint Planning Meetings in CSC301_ sections.